### PR TITLE
Lint markdown files via remark

### PR DESCRIPTION
**Pull request by ybiquitous**
See also:
- ~~https://github.com/remarkjs/remark-lint~~ (unused in this PR)
- https://github.com/remarkjs/remark-validate-links

### CODE REVIEW

**Comment by ybiquitous** [comment]
I try it, then many issues are reported. 😓 

```
$ yarn run lint
README.md: no issues found
SUMMARY.md
      4:3-4:49  warning  Link to unknown file: `getting-started/README.md`                                                                                   missing-file  remark-validate-links
      5:5-5:51  warning  Link to unknown file: `getting-started/setup.md`. Did you mean `docs/getting-started/setup.md`                                      missing-file  remark-validate-links
      6:5-6:68  warning  Link to unknown file: `getting-started/repository-settings.md`. Did you mean `docs/getting-started/repository-settings.md`          missing-file  remark-validate-links
      7:5-7:85  warning  Link to unknown file: `getting-started/custom-configuration.md`. Did you mean `docs/getting-started/custom-configuration.md`        missing-file  remark-validate-links
      8:5-8:68  warning  Link to unknown file: `getting-started/working-with-issues.md`. Did you mean `docs/getting-started/working-with-issues.md`          missing-file  remark-validate-links
      9:5-9:52  warning  Link to unknown file: `getting-started/permissions.md`. Did you mean `docs/getting-started/permissions.md`                          missing-file  remark-validate-links
    10:3-10:38  warning  Link to unknown file: `tools/README.md`                                                                                             missing-file  remark-validate-links
    11:5-11:35  warning  Link to unknown file: `tools/ruby/README.md`                                                                                        missing-file  remark-validate-links
    12:7-12:41  warning  Link to unknown file: `tools/ruby/rubocop.md`. Did you mean `docs/tools/ruby/rubocop.md`                                            missing-file  remark-validate-links
    13:7-13:35  warning  Link to unknown file: `tools/ruby/reek.md`. Did you mean `docs/tools/ruby/reek.md`                                                  missing-file  remark-validate-links
    14:7-14:39  warning  Link to unknown file: `tools/ruby/querly.md`. Did you mean `docs/tools/ruby/querly.md`                                              missing-file  remark-validate-links
    15:7-15:43  warning  Link to unknown file: `tools/ruby/brakeman.md`. Did you mean `docs/tools/ruby/brakeman.md`                                          missing-file  remark-validate-links
    16:7-16:66  warning  Link to unknown file: `tools/ruby/rails-bestpractices.md`. Did you mean `docs/tools/ruby/rails-bestpractices.md`                    missing-file  remark-validate-links
...
```

File: package.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "scripts": {
+    "lint": "remark . --frail"
```
Comment (ybiquitous) [inline package.json]: [note] `remark --help`:

```
-f  --frail               exit with 1 on warnings
```

**Comment by akito19** [comment]
> SUMMARY.md

This is *wasuregatami* of GitBook, so could you delete this file?

**Comment by ybiquitous** [comment]
OK! 👍

**Comment by ybiquitous** [comment]
@akito19 Are these lint warnings above false positives?

**Comment by akito19** [comment]
Yes. That lint seemed to require accurate file names with extension, *.md*. However, without extensions, Docusaurus can render pages.
I believe these were ignorable warnings.

**Comment by ybiquitous** [comment]
Hum... I try changing the doc below, then it works on my local machine. 🎉 

```diff
diff --git a/docs/introduction.md b/docs/introduction.md
index d71fdce..7cb61cf 100644
--- a/docs/introduction.md
+++ b/docs/introduction.md
@@ -13,7 +13,7 @@ We have prepared contents to use Sider confortably.
 
 This section has listed documents about how to use Sider.
 
-* [Setting up Sider](./getting-started/setup)
+* [Setting up Sider](./getting-started/setup.md)
 * [Repository Settings](./getting-started/repository-settings)
 * [Custom Configuration](./getting-started/custom-configuration)
 * [Working with Issues](./getting-started/working-with-issues)
```

image

---

In addition, when seeing it on GitHub, the link is broken. 🤷‍♂️ 
https://github.com/sider/sider-docs/blob/master/docs/introduction.md

image

Thus, I think better to fix the links as remark messages say, what do you think about it?

**Comment by ybiquitous** [comment]
Fixing asset URL works as well on my local machine! 🎉 

```diff
diff --git a/docs/billing-and-plans.md b/docs/billing-and-plans.md
index a8323b0..0cae942 100644
--- a/docs/billing-and-plans.md
+++ b/docs/billing-and-plans.md
@@ -14,7 +14,7 @@ Sider's plan depends on the number of users who are assigned seats. The seat-ass
 
 You can upgrade the plan and increase/decrease seats in the organization settings page.
 
-![Seat Allocation](../assets/seat_allocation_modal.png)
+![Seat Allocation](./assets/seat_allocation_modal.png)
 
 You must have the owner role for your organization if you would like to change organization settings or repository settings.
```

**Comment by akito19** [comment]
🤔 
I've never faced with 404 when clicking that link...

**Comment by ybiquitous** [comment]
What? I got 404... 🤔 

[Image from Gyazo](https://gyazo.com/c29563e7d5bac84c2b1061e731202aab)

**Comment by akito19** [comment]
Ah, I got it.
I didn't assume users read this doc on GitHub. In https://help.sider.review, all users mustn't be faced that page, so I didn't keep in mind.

If users can view docs in our domain, not GitHub.com, let's fix them 🙏

**Comment by ybiquitous** [comment]
Yeah, anyway this PR is very useful to lint on each local machine. Let's try!

**Pull request merged**